content/en/docs/collector/deployment/best-practices.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Best Practices
+description: How to configure collectors for various use cases
+weight: 4
+---
+
+Now that you are equipped with the essential deployment patterns for the
+collector, let's have a closer look at best practices for collector (pipeline)
+configurations for different use cases.
+
+## Fan Out
+
+Export signals into more than one backend. For example, for testing, policy
+(signals from dev/test go into backend X, prod goes into Y), or migrations from
+one back-end to another (cut-over) or different signal types go into different
+backends.
+
+<!-- prettier-ignore-start -->
+{{< tabpane persistLang=false >}}
+{{< tab header="Per type" lang="yaml" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
```

Comment:
This and other OTLP receiver configurations should have an explicit endpoint set to avoid a warning regarding potential DoS attacks

Reply:
As agreed, removing the best practices for this PR. Will address in the next PR.